rapid7_insightidr/help.md

```diff
@@ -36,6 +36,63 @@ Example input:
 
 ### Actions
 
+#### Query
```

Comment:
Consider renaming this to "Get Query Results." I don't think Query is the right name for this.

Reply:
I add this title same as was in JIRA. I will add this comment to JIRA for Claire.